doc/mid-based-signalling.md

```diff
@@ -0,0 +1,276 @@
+# MID Based Signalling
```

Comment:
This might be nice opportunity to fix conf vs call id somehow

Reply:
Yes, definitely. afaik its just the MSC that is confused somewhere, but note to check.